x-pack/plugins/ml/server/shared.ts

```diff
@@ -5,3 +5,4 @@
  */
 
 export * from '../common/types/anomalies';
+export { Job } from '../common/types/anomaly_detection_jobs';
```

Comment:
This could also be `*` as we might has well share all of our anomaly detection job types for future use.